Mathlib/Topology/Algebra/Module/StrongTopology.lean

```diff
@@ -247,56 +293,45 @@ variable {𝕜 : Type _} {𝕜₂ : Type _} {𝕜₃ : Type _} {𝕜₄ : Type _
     -- given explicitly to help `simps`
     invFun := fun L => (e₄₃.symm : G →SL[σ₃₄] H).comp (L.comp (e₁₂ : E →SL[σ₁₂] F))
     map_add' := fun f g => by simp only [add_comp, comp_add]
-    map_smul' := fun t f => by simp only [smul_comp, comp_smulₛₗ] }
-#align continuous_linear_equiv.arrow_congrₛₗ ContinuousLinearEquiv.arrowCongrₛₗ
-#align continuous_linear_equiv.arrow_congrₛₗ_apply ContinuousLinearEquiv.arrowCongrₛₗ_apply
-#align continuous_linear_equiv.arrow_congrₛₗ_symm_apply ContinuousLinearEquiv.arrowCongrₛₗ_symm_apply
-
-variable [RingHomIsometric σ₂₁]
-
-theorem arrowCongrₛₗ_continuous (e₁₂ : E ≃SL[σ₁₂] F) (e₄₃ : H ≃SL[σ₄₃] G) :
-    Continuous (id (e₁₂.arrowCongrₛₗ e₄₃ : (E →SL[σ₁₄] H) ≃ₛₗ[σ₄₃] F →SL[σ₂₃] G)) := by
-  apply continuous_of_continuousAt_zero
-  show Filter.Tendsto _ _ _
-  simp_rw [(arrowCongrₛₗ e₁₂ e₄₃).map_zero]
-  rw [ContinuousLinearMap.hasBasis_nhds_zero.tendsto_iff ContinuousLinearMap.hasBasis_nhds_zero]
-  rintro ⟨sF, sG⟩ ⟨h1 : Bornology.IsVonNBounded 𝕜₂ sF, h2 : sG ∈ nhds (0 : G)⟩
-  dsimp
-  refine' ⟨(e₁₂.symm '' sF, e₄₃ ⁻¹' sG), ⟨h1.image (e₁₂.symm : F →SL[σ₂₁] E), _⟩, fun _ h _ hx =>
-    h _ (Set.mem_image_of_mem _ hx)⟩
-  apply e₄₃.continuous.continuousAt
-  simpa using h2
-#align continuous_linear_equiv.arrow_congrₛₗ_continuous ContinuousLinearEquiv.arrowCongrₛₗ_continuous
-
-variable [RingHomIsometric σ₁₂]
-
-/-- A pair of continuous (semi)linear equivalences generates a continuous (semi)linear equivalence
-between the spaces of continuous (semi)linear maps. -/
-@[simps! apply symm_apply toLinearEquiv]
-def arrowCongrSL (e₁₂ : E ≃SL[σ₁₂] F) (e₄₃ : H ≃SL[σ₄₃] G) : (E →SL[σ₁₄] H) ≃SL[σ₄₃] F →SL[σ₂₃] G :=
-  { e₁₂.arrowCongrₛₗ e₄₃ with
-    continuous_toFun := e₁₂.arrowCongrₛₗ_continuous e₄₃
-    continuous_invFun := e₁₂.symm.arrowCongrₛₗ_continuous e₄₃.symm }
+    map_smul' := fun t f => by simp only [smul_comp, comp_smulₛₗ]
+    continuous_toFun := ((postcomp F e₄₃.toContinuousLinearMap).comp
+      (precomp H e₁₂.symm.toContinuousLinearMap)).continuous
+    continuous_invFun := ((precomp H e₁₂.toContinuousLinearMap).comp
+      (postcomp F e₄₃.symm.toContinuousLinearMap)).continuous }
 set_option linter.uppercaseLean3 false in
 #align continuous_linear_equiv.arrow_congrSL ContinuousLinearEquiv.arrowCongrSL
 set_option linter.uppercaseLean3 false in
 #align continuous_linear_equiv.arrow_congrSL_apply ContinuousLinearEquiv.arrowCongrSL_apply
 set_option linter.uppercaseLean3 false in
-#align continuous_linear_equiv.arrow_congrSL_symm_apply ContinuousLinearEquiv.arrowCongrSL_symm_apply
+#noalign continuous_linear_equiv.arrow_congrSL_symm_apply
+
+-- Porting note: the following two lemmas were autogenerated by `simps` in Lean3, but this is
+-- no longer the case. The first one can already be proven by `simp` so we `#noalign` it, but we
+-- state the second one manually.
+set_option linter.uppercaseLean3 false in
+#noalign continuous_linear_equiv.arrow_congrSL_to_linear_equiv
```

Comment:
You're missing `_apply`. Also, I think that we should have this lemma anyway, just without a `@[simp]` attribute.